csrc/device_lower/pass/unroll.cpp

```diff
@@ -67,9 +67,9 @@ void UnrollPass::dispatch(Expr* expr) {
   if (ir_utils::isTvOp(expr)) {
```

Comment:
can change to `if (ir_utils::isTvOp(expr) && !ir_utils::isCpAsyncBulk(expr))` to avoid unused call to `const auto out_tv = ir_utils::getTvOutput(expr);` if `ir_utils::isCpAsyncBulk(expr) == true`

Reply:
fixed